client/src/components/admin/teams/TeamsView.jsx

```diff
@@ -75,15 +81,28 @@ class TeamsView extends Component {
         e.isChecked = false;
       }
     });
-    this.setState({ cohorts: newCohortList });
+    const selectedCohort = newCohortList.filter(c => c.isChecked);
+    this.setState({
+      cohorts: newCohortList,
+      selectedCohort:
+        selectedCohort.length && selectedCohort[0].name ? selectedCohort[0].name.toLowerCase() : ''
+    });
   }
 
   showDetails() {
-    console.log('TODO: display list of students for the selected cohort');
+    const { selectedCohort, studentsListByCohort } = this.state;
+    const selectedCohortDetails = studentsListByCohort.filter(
+      cohort => cohort.name === selectedCohort
+    );
+    if (selectedCohortDetails && selectedCohortDetails.length) {
+      this.setState({
+        studentsListForSelectedCohort: selectedCohortDetails[0].students
```

Comment:
Why is selectedCohortDetails an array?

Reply:
updated to use .find instead of .filer to return an object instead of array